docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,202 @@
+# S3 Migration Guide
+
+# Upgrading to 1.0.0
+
+## Schema Changes
+
+This version introduces changes to the schema of data written to S3, which make it isomorphic to our [V2 certified database destinations](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#what-is-destinations-v2). The schema changes are as follows:
+
+| Old field name          | New field name           | Json Type          | Avro Type                                                 | Parquet Type                                 | Description                                                                                            |
+|-------------------------|--------------------------|--------------------|-----------------------------------------------------------|----------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| `_airbyte_ab_id`        | `_airbyte_raw_id`        | string (UUID)      | `{ "type": "string", "logicalType": "uuid" }`             | `String`                                     | Airbyte-added unique identifier.                                                                       |
+| `_airbyte_emitted_at`   | `_airbyte_extracted_at`  | integer (epoch ms) | `{ "type": "string", "logicalType": "timestamp-millis" }` | `Int64(logicalType=Timestamp(Milliseconds))` | Time at which the data was loaded from the source.                                                     |
+| [NEW FIELD]             | `_airbyte_generation_id` | integer            | `{ "type": "long" }`                                      | `Int64`                                      | Monotonically-increasing refresh id, if applicable.                                                    |
+| [NEW FIELD]             | `_airbyte_meta`          | object (see below) | Record (see below)                                        | Record (see below)                           | Additional metadata, including change data capture info and sync id.                                   |
+| `_airbyte_data`         | [UNCHANGED]              | optional object    | (from client schema)                                      | (from client schema)                         | Data payload when [flattening is disabled](https://docs.airbyte.com/integrations/destinations/s3#csv). |
+
+The `_airbyte_meta` field is an object that currently has one field:
+
+| Field Name     | Json Type | Description                                                    |
+|----------------|-----------|----------------------------------------------------------------|
+| `changes`      | list      | Change data capture info                                       |
```

Comment:
let's not use the literal phrase "change data capture", since it's entirely unrelated to database CDC
```suggestion
| `changes`      | list      | Modifications to the data to accomodate it for storage in S3                                       |
```

Reply:
+1, as "CDC" is a term-of-art in the data moving sector

Reply:
```A record of any changes airbyte made to the data for compatibility and/or to handle errors```